### PR TITLE
Pull request with 3 comments and 3 review comments

### PR DESCRIPTION
**Pull request by pipcrispy**

### CODE REVIEW

**Comment by pipcrispy** [comment]
comment 1

**Comment by pipcrispy** [comment]
comment 2

**Comment by pipcrispy** [comment]
comment 3

**Review by gm3dmo — COMMENTED** [review]
review comment 1

File: README.md [hunk 1]
```diff
@@ -3,3 +3,4 @@ test
 sdfsd
 dsfsdafdsfadasf
 sdfasdfsa
```
Comment (gm3dmo) [inline README.md:5]: review comment 2

File: README.md [hunk 2]
```diff
@@ -3,3 +3,4 @@ test
 sdfsd
```
Comment (gm3dmo) [inline README.md:3]: review comment 3

**Pull request merged**